modules/govuk/manifests/apps/service_manual_frontend.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: govuk::apps::service_manual_frontend
+#
+# Service Manual Frontend is an app to serve content pages for the service manual from the content store
+#
+# === Parameters
+#
+# [*vhost*]
+#   Virtual host used by the application.
+#   Default: 'service-manual-frontend'
+#
+# [*port*]
+#   What port should the app run on?
+#
+class govuk::apps::service_manual_frontend(
+  $vhost = 'service-manual-frontend',
+  $port = 3122,
+) {
+  Govuk::App::Envvar {
+    app => 'service-manual-frontend',
+  }
+
+  govuk::app { 'service-manual-frontend':
+    app_type              => 'rack',
+    port                  => $port,
+    vhost_ssl_only        => true,
+    health_check_path     => '/healthcheck',
+    legacy_logging        => false,
+    asset_pipeline        => true,
+    asset_pipeline_prefix => 'service-manual-frontend',
+    vhost                 => $vhost,
+  }
+}
```

Comment:
For new apps we should be using [12factor](http://12factor.net/) approach to configuration, where we use ENV vars to popular secret values ([multipage frontend](https://github.com/alphagov/govuk-puppet/blob/fd51ac0c86f9c481650130d8b21c155d041e5b8a/modules/govuk/manifests/apps/multipage_frontend.pp#L40-L47)'s puppet config is a good example, and [how it consumes the values](https://github.com/alphagov/multipage-frontend/blob/ee3bf30b4bdf6aa117fde8f40edeec70a6058b1c/config/initializers/airbrake.rb)). 

The values for the ENV vars come from a private configuration repo, which are injected in to the app when puppet is run - this is the private repo someone will need to help you change, as you won't have access.

`government-frontend` actually predates this approach, and these values are inserted by overwriting initialiser files - this isn't something we should copy for new apps. Could you update the puppet cdefinition to use the same pattern as `multipage-frontend`?